.github/workflows/python-test.yaml

```diff
@@ -0,0 +1,237 @@
+name: Python Test
+
+on:
+  workflow_run:
+    workflows: [Python Build]
+    types:
+      - completed
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-latest
+            target: x86_64
+          - runner: ubuntu-latest
+            target: x86
+          - runner: ubuntu-latest
+            target: aarch64
+          - runner: ubuntu-latest
+            target: armv7
+          - runner: ubuntu-latest
+            target: s390x
+          - runner: ubuntu-latest
+            target: ppc64le
+        nox-session:
+          # To speed things up a bit we use the special ci_checks_max session
+          # that uses the same venv to run multiple linting sessions
+          - "ci_checks_max"
+          - "pytest_min"
+        python:
+          - "3.11"
+          - "3.12"
+    steps:
```

Comment:
So, you couldn't use the GH action for nox? https://github.com/frequenz-floss/gh-action-nox

If that the case it would be interesting to see if we can adapt it to support this workflow too, to avoid duplication as much as possible.

Reply:
Installing the wheel inside the nox env requires me to use it this way, I guess. Or can I somehow hook into the action to install a local wheel?

Reply:
Oh, OK, you are installing from the built wheel. Yes, we could potentially add an option to the reusable action to pass an optional wheel to install instead of doing `pip install .`.

Reply:
Unresolving until we decide what to do, and create an issue in the GH action if we want to go in that direction.

Reply:
Issue about it:
* https://github.com/frequenz-floss/gh-action-nox/issues/8